tests/unit/s2n_key_update_test.c

```diff
@@ -156,90 +155,108 @@ int main(int argc, char **argv)
             client_conn->actual_protocol_version = S2N_TLS13;
             client_conn->secure.cipher_suite = &s2n_tls13_aes_256_gcm_sha384;
             memcpy_check(client_conn->secure.client_app_secret, application_secret.data, application_secret.size);
+            uint8_t expected_sequence_number[S2N_TLS_SEQUENCE_NUM_LEN] = {0};
 
-            uint8_t data_size = 1;
-            client_conn->secure.client_sequence_number[0] = 1; 
             client_conn->key_update_pending = false;
-            client_conn->encrypted_bytes_out = S2N_TLS13_AES_GCM_MAXIMUM_BYTES_TO_ENCRYPT;
 
-            EXPECT_SUCCESS(s2n_key_update_send(client_conn, data_size));
+            /* set record number to encryption limit */ 
+            client_conn->secure.client_sequence_number[4] = 1;
+            client_conn->secure.client_sequence_number[5] = 106;
+            client_conn->secure.client_sequence_number[6] = 9;
+            client_conn->secure.client_sequence_number[7] = 229;
```

Comment:
23726565 = { 0x1, 0x6A, 0x09, 0xE5 }. 

numbers looks good to me here, perhaps update the comments so someone reading this could understand the numbers?